ansible_collections/arista/avd/roles/eos_l3ls_evpn/README.md

```diff
@@ -926,6 +926,12 @@ l3_edge:
       # QOS Service Profile | Optional
       qos_profile: < qos_profile_name >
 
+      # MACSec Profile | Optional
+      macsec_profile: < macsec_profile_name >
```

Comment:
Seems MACSEC profile definition is not in readme file

Reply:
@titom73 The actual `macsec_profiles` are only available in `eos_cli_config_gen` data model, but here we just need to reference it. There is currently no need to abstract `macsec_profiles` in `eos_design`.

Reply:
Ack thanks